frontend/src/sass/layout/_header.scss

```diff
@@ -1,6 +1,42 @@
 /* =================================
    LAYOUT / HEADER
    ================================= */
+.header {
+	background-color: $purple;
+	a {
+        color: $white;
+		display: inline-block;
+    }
+    .content-wrap {
+    	display: flex;
+    	flex-direction: row;
+    	flex-wrap: wrap;
+		align-items: center;
+		justify-content: center;
```

Comment:
weird it doesn't look like that locally :/ 

![screen shot 2016-06-19 at 16 24 15](https://cloud.githubusercontent.com/assets/2911613/16178232/525af37a-363a-11e6-884b-84fe5665a096.png)